src/atoms/Text/Text.stories.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import Text from './Text';
+import { withKnobs } from '@storybook/addon-ondevice-knobs';
+import { select, text, color } from '@storybook/addon-knobs';
+import fonts from '../../tokens/fonts';
+const theme = {
+  fonts,
+};
+
+const weightOptions = {
+  regular: 'regular',
+  bold: 'bold',
+};
+
+const letterSpacingOptions = {
+  s: 's',
+  m: 'm',
+  l: 'l',
+};
+
+storiesOf('Text', module)
+  .addParameters({
+    component: Text,
+  })
+  .addDecorator(withKnobs)
+  .add('default', () => (
+    <Text
+      size={select('Size', Object.keys(theme.fonts.size), 'l')}
```

Comment:
I'm a bit afraid about keeping the text component too flexible. The reason is if you refer our Figma designs they have like 5 variations of `Text` component. How about if we just have the api something like below
```jsx
<Text
type="xxs/xs/s/m/l/xxl"
color="our colors"
truncate=true/false
align="not sure if this makes sense for react native"
as="p/div/span - this is for web mostly but api should remain consistent"
</Text>
```
All the font weights, sizes let them be the implementation details.

Thoughts?

Reply:
`xxs/xs/s/m/l/xxl` are variants for font size. Weights are still independent of this. There is a `H6`  & `T2` with font size as `m` but font-weight as `bold` & `regular`. How do we interpret weight based on solely the size?

Reply:
It depends on what `align' does on web. On react-native `align-self` is something that we can probably use often which is why I exposed that. We could probably have a single `align` that behaves platform specific